examples/oneapi/cpp/source/decision_forest/df_cls_dense_batch.cpp

```diff
@@ -56,22 +56,18 @@ int main(int argc, char const *argv[]) {
           .set_features_per_node(1)
           .set_min_observations_in_leaf_node(1)
           .set_variable_importance_mode(df::variable_importance_mode::mdi)
-          .set_train_results_to_compute(
-              df::train_result_to_compute::compute_out_of_bag_error)
-          .set_infer_results_to_compute(
-              df::infer_result_to_compute::compute_class_labels |
-              df::infer_result_to_compute::compute_class_probabilities)
           .set_voting_method(df::voting_method::weighted);
 
-  const auto result_train = dal::train(df_desc, x_train_table, y_train_table);
+  const auto result_train = dal::train(df_desc, x_train_table, y_train_table, df::train_result_to_compute::compute_out_of_bag_error);
 
   std::cout << "Variable importance results:" << std::endl
             << result_train.get_var_importance() << std::endl;
 
   std::cout << "OOB error: " << result_train.get_oob_err() << std::endl;
 
   const auto result_infer =
-      dal::infer(df_desc, result_train.get_model(), x_test_table);
+      dal::infer(df_desc, result_train.get_model(), x_test_table, df::infer_result_to_compute::compute_class_labels |
```

Comment:
Seems like line wrapping convention is different for classification and regression examples